plugins/@grouparoo/iterable/__tests__/export/export-profile.ts

```diff
@@ -529,6 +529,65 @@ describe("iterable/exportProfile", () => {
     expect(brandNewUser.dataFields.name).toBe(brandNewName);
   });
 
+  test("can update the new user on email change if both emails exist", async () => {
+    await runExport({
+      oldProfileProperties: {
+        email: brandNewEmail,
+        userId: brandNewUserId,
+        name: brandNewName,
+      },
+      newProfileProperties: {
+        email: newEmail,
+        userId: newUserId,
+        name: otherName,
+      },
```

Comment:
So... `newEmail` is newer than `brandNewEmail`?   This can probably be tightened up a bit :D

Reply:
Hahahah true! I just reused names that were there. Cleaned it up a bit and changed the ordering so mental gymnastics are kept to a minimum